compiler/qsc_parse/src/stmt/tests.rs

```diff
@@ -603,18 +600,19 @@ fn doc_followed_by_non_item() {
         }",
         &expect![[r#"
             Block _id_ [0-72]:
-                Stmt _id_ [14-62]: Err
+                Stmt _id_ [14-39]: Item: Item _id_ [0-0]:
+                    Err
+                Stmt _id_ [52-62]: Local (Immutable):
+                    Pat _id_ [56-57]: Bind:
+                        Ident _id_ [56-57] "x"
+                    Expr _id_ [60-61]: Lit: Int(2)
```

Comment:
This is kind of a nice side effect of your fix :) . We recover enough to be able to parse the `let` statement now. That should improve both error messages and language server features for imperfect code.